library/core/src/ptr/const_ptr.rs

```diff
@@ -520,9 +520,15 @@ impl<T: ?Sized> *const T {
     /// Calculates the distance between two pointers. The returned value is in
     /// units of T: the distance in bytes is divided by `mem::size_of::<T>()`.
     ///
-    /// This function is the inverse of [`offset`].
+    /// This is equivalent to `(self as isize - other as isize) / mem::size_of::<T>()`.
```

Comment:
Equivalent, modulo UB. So I think we shouldn't just say "equivalent" without further qualification.